metron-analytics/metron-profiler/src/main/java/org/apache/metron/profiler/bolt/ProfileHBaseMapper.java

```diff
@@ -49,7 +65,7 @@
 
   public ProfileHBaseMapper() {
     setColumnFamily("P");
-    setSaltDivisor(1000);
+    setSaltDivisor(0);
```

Comment:
Why are you setting the salt divisor to 0?  Under what situations would you not need a salt?  I'm especially concerned that this appears to be the default behavior.